assignment-2/src/view/QuestionList.js

```diff
@@ -0,0 +1,89 @@
+import React from "react"
+
+const QuestionList = ({questions,onSearchTitle, newQuestion, 
```

Comment:
This component is pretty big (hence also the large number of parameters). Split it into multiple components.